app/manage/pods/rules-form/conditional-form-directive/template.tmpl

```diff
@@ -47,8 +47,10 @@
         <i class="fa fa-pencil" ng-click='editCondition()'></i>
         <i class="fa fa-times" ng-click='deleteCondition(index)'></i>
       </span>
+      <!--edit mode -->
+      <button class='button secondary' ng-if="!listMode && editting" ng-click='editCondition()'>+ Save </button>
```

Comment:
ohhh I didn't even notice that. Perhaps it should just make the server call? which would mean no more than 1 could be active at a time.. I can see somebody clicking save and navigating away quickly